samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
There was no need to have done that. This is the sort of in-the-wild discrepancy that's both natural and expected; and also part of the reason we need samples in the first place.